### DISCUSSION TITLE
Electrotine ore should be a part of Fabrication

### DISCUSSION THREAD

**Discussion by ThisIsSP4RTAN** [MrTJP/ProjectRed#1823] [Ideas]
As the title states, Electrotine ore should probably be added by Fabrication instead of Exploration even though it's an ore (or add a crafting recipe for it if you don't have Exploration installed). If you install Fabrication without Exploration there's no way to obtain Electrotine dust which you need in order to craft anything in Fabrication.

**Comment by ThisIsSP4RTAN** [comment]
Or I guess Exploration could just be marked as another requirement of Fabrication, but I'm not a huge fan of being forced to use another module if I don't want to just because of one missing thing. For now though, I just added a basic shapeless crafting recipe to create Electrotine dust.

**Comment by MrTJP** [comment]
If you don't want to install Exploration for the world gen, you can install other mods that help you acquire what you need in other ways (as you have already done). It doesn't make sense to mark Exploration as a requirement as there is no code dependency that requires Exploration for Fabrication to run properly.
